src/app/mail/mail-sidebar/compose-mail-dialog/compose-mail-dialog.component.ts

```diff
@@ -50,6 +51,7 @@ export class ComposeMailDialogComponent implements OnInit, AfterViewInit {
     if (this.draft) {
       this.mailSubject = this.draft.subject;
     }
+    this.isFullScreen = this.is_composer_full_screen;
```

Comment:
why 2 variables for same purpose?
`this.isFullScreen = this.is_composer_full_screen `

Reply:
sorry, it was not necessary, let me update.